packages/lesswrong/server/recombee/client.ts

```diff
@@ -447,7 +447,7 @@ const recombeeApi = {
 
     let deferredPostsPromise: Promise<DbPost[]> | undefined = undefined;
     let recombeeResponsesWithScenario;
-    if (lwAlgoSettings.hybridScenarios.fixed === 'recombee-emulate-hacker-news') {
+    if (lwAlgoSettings.hybridScenarios.fixed === 'lesswrong-hacker-news') {
```

Comment:
If we're changing this, shouldn't it be to `forum-classic`?

Reply:
Yeah, that makes sense